src/effects/native/balanceeffect.cpp

```diff
@@ -1,5 +1,9 @@
 #include "balanceeffect.h"
 
+namespace {
```

Comment:
So are you proposing that the input signal to the effect would be split and only the higher frequencies would be effected, then the low frequencies would be mixed back in after the effect adjusts the balance of the higher frequencies?